lib/weblateLanguages/languages.json

```diff
@@ -0,0 +1 @@
orthern Sami","sef":"Senoufo (Cebaara)","seh":"Sena","sel":"Selkup","ses":"Koyraboro Senni","sg":"Sango","sga":"Old Irish","sgn":"Sign Languages","sgs":"Samogitian","shi":"Tachelhit","shn":"Shan","si":"Sinhala","sid":"Sidamo","sjd":"Sami (Kildin)","sk":"Slovak","skr":"Saraiki","sl":"Slovenian","sm":"Samoan","sma":"Southern Sami","smi":"Sami","smj":"Lule Sami","sml":"Sama (Central)","smn":"Inari Sami","sms":"Skolt Sami","sn":"Shona","snk":"Soninke","so":"Somali","sog":"Sogdien","son":"Songhai","sou":"Thai (Southern)","sq":"Albanian","sr":"Serbian","sr@ijekavian":"Serbian (Ijekavian)","sr@ijekavian_Latn":"Serbian (Ijekavian, latin)","sr_Cyrl":"Serbian (Cyrillic)","sr_Latn":"Serbian (Latin)","srn":"Sranan Tongo","srr":"Serer","ss":"Swati","ssy":"Saho","st":"Southern Sotho","su":"Sundanese","suk":"Sukuma","sus":"Susu","sux":"Sumerian","sv":"Swedish","sw":"Swahili","sw_CD":"Swahili (Congo - Kinshasa)","sw_TZ":"Swahili (Tanzania)","swg":"Swabian","swv":"Shekhawati","sxu":"Saxon (Upper)","syc":"Classical Syriac","syl":"Sylheti","syr":"Syriac","szl":"Silesian","ta":"Tamil","ta_LK":"Tamil (Sri Lanka)","tcy":"Tulu","te":"Telugu","tem":"Timne","teo":"Teso","ter":"Tereno","tet":"Tetum","tg":"Tajik","th":"Thai","ti":"Tigrinya","tig":"Tigre","tiv":"Tiv","tk":"Turkmen","tkl":"Tokelau","tl":"Tagalog","tlh":"Klingon","tlh-qaak":"Klingon (Qaak)","tli":"Tlingit","tly":"Talysh","tmh":"Tamashek","tn":"Tswana","to":"Tongan","tog":"Nyasa Tonga","tok":"Toki Pona","tpi":"Tok Pisin","tr":"Turkish","trv":"Taroko","ts":"Tsonga","tsg":"Tausug","tsi":"Tsimshian","tsj":"Tshangla","tt":"Tatar","tt@iqtelif":"Tatar (IQTElif)","tts":"Thai (Northeastern)","tum":"Tumbuka","tvl":"Tuvalu","tw":"Twi","ty":"Tahitian","tyv":"Tuvinian","tzj":"Tz'utujil","tzl":"Talossan","tzm":"Central Atlas Tamazight","udm":"Udmurt","ug":"Uyghur","uga":"Ugaritic","uk":"Ukrainian","umb":"Umbundu","und":"Unknown language","unr":"Mundari","ur":"Urdu","ur_IN":"Urdu (India)","ur_PK":"Urdu (Pakistan)","uz":"Uzbek","uz_Latn":"Uzbek (Latin)","vai":"Vai","ve":"Venda","vec":"Venetian","vi":"Vietnamese","vls":"West Flemish","vmf":"Main-Franconian","vmw":"Makhuwa","vo":"Volap\u00fck","vot":"Votic","vro":"V\u00f5ro","vun":"Vunjo","wa":"Walloon","wae":"Walser","wal":"Wolaytta","war":"Waray","was":"Washo","wbq":"Waddar","wbr":"Wagdi","wen":"Sorbian","wep":"Westphalien","wo":"Wolof","wtm":"Mewati","wuu":"Wu Chinese","wuu_Hans":"Wu Chinese (Simplified)","wuu_Hant":"Wu Chinese (Traditional)","xal":"Kalmyk","xh":"Xhosa","xnr":"Kangri","xog":"Soga","yao":"Yao","yap":"Yapese","yi":"Yiddish","yo":"Yoruba","yua":"Yucateco","yue_Hans":"Cantonese (Simplified)","yue_Hant":"Cantonese (Traditional)","za":"Zhuang","zap":"Zapotec","zbl":"Blissymbols","zen":"Zenaga","zgh":"Standard Moroccan Tamazight","zh_Hans":"Chinese (Simplified)","zh_Hans_SG":"Chinese (Simplified, Singapore)","zh_Hant":"Chinese (Traditional)","zh_Hant_HK":"Chinese (Traditional, Hong Kong SAR China)","zh_Latn":"Chinese (Latin)","zu":"Zulu","zun":"Zuni","zza":"Zaza"}
```

Comment:
Where this is coming from? :-)

Reply:
I'll undo this one.. this I don't think I should've committed this 🤔

Reply:
This is now removed. I accidentally included it when I was fixing some conflicts during rebase.